data/release/security/csp.xml

```diff
@@ -0,0 +1,95 @@
+<config>
```

Comment:
oh no I must remove these files from running locally

Reply:
it was going to be my first comment